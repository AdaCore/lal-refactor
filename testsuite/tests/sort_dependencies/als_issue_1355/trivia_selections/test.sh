#!/bin/sh

sort_dependencies -P default.gpr -S main.adb --start-line 1 --start-column 1 --end-line 8 --end-column 1 --no-separator
