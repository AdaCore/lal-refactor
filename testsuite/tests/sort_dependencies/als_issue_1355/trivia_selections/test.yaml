driver: shell_script
title: Selection start and end on trivia
description: sort_dependencies tool test
