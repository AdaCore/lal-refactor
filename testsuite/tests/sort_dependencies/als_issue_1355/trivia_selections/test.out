main.adb
3:1-7:19 with Ada.Calendar;
with Ada.Command_Line;
with Ada.Integer_Text_IO, Ada.Text_IO, Ada.Float_Text_IO;
with Ada.Numerics.Elementary_Functions;

